Source/zerochecker.h

```diff
@@ -19,12 +19,16 @@ namespace zero
     // Contains first and last non-zero data for a given audio file
     struct File
     {
-		juce::File m_file{};
+        File(const juce::File file);
```

Comment:
Any reason you're passing this by value? I'd probably do this to avoid the unnecessary copy.
```
File(const juce::File& file);
```